src/main/java/li/cil/tis3d/common/api/FontRendererAPIImpl.java

```diff
@@ -1,9 +1,14 @@
 package li.cil.tis3d.common.api;
```

Comment:
Weird nitpick, but wouldn't this file technically belong in `tis3d.client.api`?

Reply:
Yes. Definitely. No idea how it got there.